mantle/platform/qemu.go

```diff
@@ -255,6 +255,17 @@ func (inst *QemuInstance) WaitIgnitionError(ctx context.Context) (string, error)
 	return r.String(), nil
 }
 
+// Read file, output contents and return error if reading fails
+func (inst *QemuInstance) ReadFile(filename string) (string, error) {
+	data, err := os.ReadFile(filename)
+	errchan := make(chan error)
+	if err != nil {
+		errchan <- err
+		return "", err
+	}
+	return string(data), nil
+}
```

Comment:
this function isn't needed.. just use `os.Readfile()` like is done here:

https://github.com/coreos/coreos-assembler/blob/7550a58bb623f9cf2c754dc130516ca394a4eec5/mantle/cmd/kola/console.go#L70

Reply:
Make sense...